tests/test_prefect_helpers.py

```diff
@@ -15,8 +23,6 @@ def test_enable_loguru_support():
     error, though whether it still works is a completely different
     question!"""
 
-    @flow
-    def example_flow():
-        enable_loguru_support()
-
-    example_flow()
+    with prefect_test_harness():
```

Comment:
Can we put this on one line?
```suggestion
    with prefect_test_harness(), disable_run_logger()
```